tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "noUnusedParameters": true,
     "strictFunctionTypes": true,
     "esModuleInterop":true,
-    "allowSyntheticDefaultImports": true, 
+    "allowSyntheticDefaultImports": true,
+    "allowImportingTsExtensions": true
```

Comment:
curious as to what why `"allowImportingTsExtensions"` was needed for your context?

Reply:
I was getting a typescript error that said you couldn't import .tsx files which seemed odd. After researching a bit I found that this fixed the issue and I hadn't come across any horror stories as to why it might be bad so I went ahead and tried it but, I also don't know as much as I probably should about the problem lol.